about.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>informing the universe...</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+  <link rel="stylesheet" type="text/css" href="stylesheets/about-styles.css">
+</head>
+
+<body>
+	<header>
+		<nav class="header-nav">
+			<ul>
+				<li><a href="./index.html" alt="return to the home page">home</a></li>
+				<li><a href="./portfolio.html" alt="a portfolio of code projects">portfolio</a></li>
+				<li><a href="./hobby-blog.html" alt="a blog about making historical costumes">costume blog</a></li>
+			</ul>
+		</nav>
+	</header>
+
+	<section class="main-body">
+		<img src="images/rowan.jpg" alt="an illustration of Rowan with pink hair" class="author-photo" height="400" width="400">
+
+		<section class="bio">
+			<p>Currently a student at Ada Developer's Academy, in past manifestations Rowan has been a DJ, a bartender, an executive assistant, and a bookseller. She's currently tackling learning a whole stack of new technologies (see, that's a pun, and she nods at her own cleverness) and writing in the third person.</p>
+
+			<p>In addtition to technological pursuits, Rowan designs tabletop and Live Action Roleplaying Games, as well as speaking at conferences for designers about the importance of inclusivity, respect, and consent as design principles. As an educator, Rowan believes she can achieve her goal of melding the worlds of design and creation to achieve works that have our core principles as people baked in from the start.</p>
+
+
+			<section class="mini-lists">
+				<h2 class="likes">Likes:</h2>
+				<p>
+					<ul>
+						<li>punctuation</li>
+						<li>else clauses</li>
+						<li>debugging</li>
+					</ul>
+				</p>
+
+				<h2 class="dislikes">Dislikes:</h2>
+				<p>
+					<ul>
+						<li>feature creep</li>
+						<li>missing close brackets</li>
+						<li>writing commit messages</li>
+						<li>unbalanced lists</li>
+					</ul>
+				</p>
+			</section>
+		</section>
+	</section>
+
+		<section class="footer-bar">
```

Comment:
Careful about your nesting here so you maintain semantic structure communicating parent-child relationships.